README_sqlite_dbs.md

```diff
@@ -7,12 +7,27 @@ This directory contains SQLite databases that make our clean tabular
 There are DBs for the versions of the MCRF OMOP CDM data, the CDM
 vocabulary, and examples for various classification tasks.
 
+### naming convention
+
+The emr DBs are named following the convention 
+'db_name.data_date.tags.sqlite3' where db_name is
+one of 'mcrf', 'emr', 'vocab', data_date (YYYYMMDD) indicates the version of 
+MCRF data based on d/l date, and tags is an extensible dotted field 
+(ex ...data_date.tag.tag.sqlite3) that can be used to provide addional
+information about the contents of the DB file.  Users wishing to share
+derived database files are encouraged to follow this format to assist with
+data version tracking and searchability. 
+
+### queries
```

Comment:
Please follow the Markdown conventions found in the other files, namely:
* Headers use titlecase (follows style in printed matter)
* 2 blank lines before, 1 after (using blank lines to separate paragraphs -> 2 blank lines between sections)
* Headers increase only 1 level (don't jump from level 1 to 3)
* Underline level 2 headers with dashes (I prefer the underline look over hashes because the visual weight of hashes is inversely proportional to the header importance)
* Level 3 headers include trailing hashes for visual balance
* Wrap paragraphs to 72 columns (Python convention for line length)

If you have strong opinions on these conventions, we can discuss them on Slack.